1.chunkedgraph.md

```diff
@@ -97,29 +105,40 @@ This method allows us to use a private cluster easily. If you need to give clust
 
 </details>
 
+Updates a kubeconfig file with appropriate credentials and endpoint information to point kubectl at a specific cluster in Google Kubernetes Engine. Then, you can select your cluster with `kubectx`.
+
+```
+gcloud container clusters get-credentials <CLUSTER_NAME> --region <REGION_NAME> --project <PROJECT_ID>
+```
+
```

Comment:
Thank you for the pr! This is repeated information, already covered by `kubernetes_cluster_context` from terraform's output; and doesn't belong in the private cluster section.

Also, users might not be familiar with `kubectx`. It could be included as an optional install in requirements.

Rest of the changes look good.